components/esp_websocket_client/examples/target/main/Kconfig.projbuild

```diff
@@ -20,6 +20,20 @@ menu "Example Configuration"
         help
             URL of websocket endpoint this example connects to and sends echo
 
+    config WS_OVER_TLS_SERVER_AUTH
```

Comment:
Could you please check if `echo.websocket.events` provides a wss option? (would `wss://echo.websocket.events` work?)

Reply:
If we find a good public endpoint which supports wws, we don't have to use self-signed certs. And it would be more straight forward demonstration